sqlobject/src/main/java/org/jdbi/v3/sqlobject/DefaultObjectBinder.java

```diff
@@ -19,20 +19,35 @@
 
 class DefaultObjectBinder implements Binder<Bind, Object>
 {
+    private final int paramIndex;
+
+    DefaultObjectBinder()
+    {
+        this(-1);
+    }
```

Comment:
yeah, if you have `@Bind(binder=DefaultObjectBinder.class)`, it calls `DefaultObjectBinder.class.newInstance()`